app/viewModels/ViewMovement.scala

```diff
@@ -19,20 +19,51 @@ package viewModels
 import java.time.format.DateTimeFormatter
 import java.time.{LocalDate, LocalTime}
 
+import config.FrontendAppConfig
+import models.{Arrival, ViewMovementAction}
+import play.api.i18n.Messages
 import play.api.libs.json.{JsObject, Json, OWrites}
 
-final case class ViewMovement(date: LocalDate, time: LocalTime, movementReferenceNumber: String, status: String)
+final case class ViewMovement(date: LocalDate, time: LocalTime, movementReferenceNumber: String, status: String, action: Seq[ViewMovementAction])
 
 object ViewMovement {
+
+  def apply(arrival: Arrival)(implicit messages: Messages, frontendAppConfig: FrontendAppConfig): ViewMovement = {
+
+    val status = arrival.status match {
```

Comment:
The more I look at this I think the status should be a separate method called within ViewMovement (similar to actions). But it works and am sure it will change as we add more status' in the future

Reply:
Done